frontend/frontend-filters-widgets.php

```diff
@@ -63,9 +63,7 @@ public function sidebars_widgets( $sidebars_widgets ) {
 
 		$sidebars_widgets = $this->filter_widgets_sidebars( $sidebars_widgets, $wp_registered_widgets );
 
-		$this->cache->set( "sidebars_widgets_{$cache_key}", $sidebars_widgets );
-
-		return $sidebars_widgets;
+		return $this->cache->set( "sidebars_widgets_{$cache_key}", $sidebars_widgets );
```

Comment:
Do you think it's a good enhancement to have a static method like `PLL_Cache::get_unique_key()`?

Reply:
Yes, I think so. I see the same thing is also used in `PLL_Cache::count_posts()`.
It's out of scope but let's see if this passes the final review 😁

Reply:
Done in [cfd7677](https://github.com/polylang/polylang/pull/1403/commits/cfd7677bbcc1147c410dc3d4330e79e23cead2d3).